packages/babel-types/package.json

```diff
@@ -17,8 +17,10 @@
   "types": "lib/index.d.ts",
   "dependencies": {
     "@babel/helper-validator-identifier": "^7.10.4",
-    "lodash": "^4.17.13",
-    "to-fast-properties": "^2.0.0"
```

Comment:
`to-fast-properties` is incorrectly removed.

Reply:
Nice catch, thanks!